pg_service_parser/conf/service_settings.py

```diff
@@ -10,8 +12,9 @@
         "description": "Password to be used if the server demands password authentication.",
     },
     "sslmode": {
-        "default": "",
+        "default": SslModeEnum.PREFER.value,
         "description": "This option determines whether or with what priority a secure SSL TCP/IP connection will be negotiated with the server.",
+        "values": SslModeEnum,
```

Comment:
We could also add tooltips for each value here, perhaps as a separate key (e.g., `"value_tooltips"`) and assign a list of tooltips. 

Now, this dict is growing as you said, we could perhaps address this in a follow-up PR later (I'll wait for your instructions regarding that).